### ISSUE TITLE
Automate sync process

### ISSUE THREAD

**Issue by w8896699** [bcgov/platform-services-registry#612]
**Describe the issue**
Currently sync needs to be manually triggered

**Additional context**
Add any other context, attachments or screenshots

**Definition of done**
Automated the whole sync process

**Comment by mitovskaol** [comment]
@w8896699 Don't have already have an automated sync process?

**Comment by w8896699** [comment]
No, we have to manually trigger them, ideally, they can have a cone job or auto-detect failure and run jobs.
